eng/common/templates/steps/send-to-helix.yml

```
@@ -0,0 +1,77 @@
+parameters:
+  HelixSource: 'pr/dotnet-github-anon-kaonashi-bot'     # required
+  HelixType: ̓'tests/default/'                           # required
+  HelixBuild: $(Build.BuildNumber)                      # required
+  HelixTargetQueues: ''                                 # required
+  HelixAccessToken: ''                                  # required
+  HelixPreCommands: ''
+  HelixPostCommands: ''
+  WorkItemDirectory: ''
+  WorkItemCommand: ''
+  CorrelationPayloadDirectory: ''
+  XUnitProjects: ''
+  XUnitTargetFramework: ''
+  XUnitRunnerVersion: ''
+  IncludeDotNetCli: false
+  DotNetCliPackageType: ''
+  DotNetCliVersion: ''
+  EnableXUnitReporter: false
+  WaitForWorkItemCompletion: true
+  condition: succeeded()
+  continueOnError: false
+
+steps:
+  - script: powershell eng/common/tools.ps1
+    displayName: Install tools (Windows)
+    condition: and(${{ parameters.condition }}, eq(variables['Agent.Os'], 'Windows_NT'))
+  - script: scriptroot="$BUILD_SOURCEDIRECTORY/eng"; source eng/common/tools.sh
```

I think this can just be
```suggestion
  - script: /bin/sh eng/common/tools.sh
```

---

big if true. will test.